src/lib/support/Span.h

```diff
@@ -89,6 +89,13 @@ class Span
         return Span(mDataBuf + offset, length);
     }
 
+    template <size_t N>
+    FixedSpan<T, N> subspan(size_t offset) const
+    {
+        VerifyOrDie(offset + N <= mDataLen);
```

Comment:
How is this guarding against offset + N overflowing, e.g. because offset is really big?

Should probably have checks more like those in `SubSpan`.